test.py

```diff
@@ -1,2 +1,5 @@
+
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test.py
+++ b/test.py
@@ -1,4 +1,3 @@
-
 
 
 def main():
```